cmd/livepeer/livepeer.go

```diff
@@ -515,9 +515,12 @@ func main() {
 				// Can't divide by 0
 				panic(fmt.Errorf("The amount of pixels per unit must be greater than 0, provided %d instead\n", *pixelsPerUnit))
 			}
-			if *pricePerUnit <= 0 {
-				// Prevent orchestrator from unknowingly provide free transcoding
-				panic(fmt.Errorf("Price per unit of pixels must be greater than 0, provided %d instead\n", *pricePerUnit))
+			if !isFlagSet["pricePerUnit"] && *pricePerUnit == 0 {
```

Comment:
Technically we don't need the `*pricePerUnit == 0` check since the default is 0. But, I figured it would be nice to have in case the default becomes non-zero in the future since we only want to error out if the flag is not set *and* the default value is 0.